s6/Makefile

```diff
@@ -3,8 +3,8 @@
 PWD=$(shell pwd)
 
 SRC=$(PWD)/../submodules
-BLD=$(PWD)/build
-RPMDIR=$(BLD)/rpms
+RPMDIR=$(PWD)/build/rpms
+BLD=$(RPMDIR)/BUILD/opt/s6
```

Comment:
Variable with obfuscated names, some with some path 'BUILD' in it, some without and you have to add it (see line 97).

Looks over-complicated for what it is. Seems ripe for a cleanup.